web/src/app/modules/views/side/modules/errors-warnings/components/errors-warnings.component.html

```diff
@@ -0,0 +1,27 @@
+<div *ngIf="visible" class="card">
+    <h5 class="card-header">
+        {{'ErrorsandWarnings' | translate}}
+        <button id="linksactions-expand-button" type="button" class="btn btn-sm btn-secondary pull-right" (click)="isCollapsed = !isCollapsed" [attr.aria-expanded]="!isCollapsed" aria-controls="collapseExample"><i *ngIf="isCollapsed" class="fa fa-angle-double-down"></i><i *ngIf="!isCollapsed" class="fa fa-angle-double-up"></i></button>
+    </h5>
+    <div class="card-body" *ngIf="!isCollapsed">
+        <ul class="list-group list-group-flush">
```

Comment:
I think a `<ul>`-Element without a nested `<li>`-Element is invalid. Additionally, it would be nicer if the table would span the whole width without any margin or padding around it.